lms/templates/header/user_dropdown.html

```diff
@@ -16,7 +16,7 @@
 self.real_user = getattr(user, 'real_user', user)
 profile_image_url = get_profile_image_urls_for_user(self.real_user)['medium']
 username = self.real_user.username
-resume_block = retrieve_last_sitewide_block_completed(username)
+resume_block = retrieve_last_sitewide_block_completed(self.real_user)
```

Comment:
I guess the confusion is around how we can change the type being sent into the method without any repercussions. 

It seems like we are going from a String here to a User object or id without making any other change.